Big change always starts with small ideas! Project 3

RebeccaBrogli opened the pull request:
I wanted to build a website that helps the reader to be in the perfect mind space to develop ideas and unleash creativity. It is a collection of videos, links, and ideas to guide the reader to create bold ideas.

I am looking forward to getting some feedback. Thank you! Rebecca

RebeccaBrogli committed:
added several pages, menu bar

RebeccaBrogli committed:
Added a button, links and About me page

RebeccaBrogli committed:
Add files via upload

Added Readme! Italic

RebeccaBrogli committed:
Delete IMG_1695.JPG

RebeccaBrogli committed:
Delete IMG_5727.jpg

RebeccaBrogli committed:
Delete 61b17f92-53a3-4bc9-bddf-f0606e35f36b.jpg

RebeccaBrogli committed:
Delete heading.jpg

RebeccaBrogli committed:
Delete katrina-wright-yMg_SMqfoRU-unsplash.jpg

RebeccaBrogli committed:
Delete rebeccawithflowers.jpg

RebeccaBrogli committed:
Delete example-recipe-background.png

RebeccaBrogli committed:
Delete diego-ph-fIq0tET6llw-unsplash.jpg

RebeccaBrogli committed:
Delete CCE99C82-87A0-4844-BE77-372764A5ABB3.png

RebeccaBrogli committed:
Add files via upload

RebeccaBrogli committed:
Added a Favicon

RebeccaBrogli committed:
Delete about.html

RebeccaBrogli committed:
Delete index.html

RebeccaBrogli committed:
Delete style.css

RebeccaBrogli committed:
Delete toolbox.html

RebeccaBrogli committed:
Delete 61b17f92-53a3-4bc9-bddf-f0606e35f36b.jpg

RebeccaBrogli committed:
Delete IMG_1695.JPG

RebeccaBrogli committed:
Delete IMG_5727.jpg

RebeccaBrogli committed:
Delete rebeccawithflowers.jpg

RebeccaBrogli committed:
Delete katrina-wright-yMg_SMqfoRU-unsplash.jpg

RebeccaBrogli committed:
Delete heading.jpg

RebeccaBrogli committed:
Started with responsive design

HIPPIEKICK commented on index.html:
```diff
@@ -0,0 +1,88 @@
+          <p> Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised ...
+          
+          <a href="toolbox.html">  
+            <button class= "btn" type="button"> 
```
Remove the space between = and the class name

HIPPIEKICK commented on index.html:
```diff
@@ -0,0 +1,88 @@
+
+    <div class="image_bottom_index">
+      <h3> Everything begins with an idea! </h3> 
+    </div>
```
Really nice looking section!

HIPPIEKICK commented on toolbox.html:
```diff
@@ -0,0 +1,70 @@
+          <source 
+          src="/Users/rebeccabrogli/Desktop/Project1/project-p1-first-webpage-master/video/video1.mp4" type="video/mp4">
+          Your browser does not support the video tag.
+        </video>  
```
Really nice with a video 🔥

HIPPIEKICK commented on style.css:
```diff
@@ -0,0 +1,398 @@
+   Reset CSS Stylesheet! 
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
```
There's a nifty trick to selecting all elements in the HTML document, just write
```
* {
  /*Code that should apply to everything*/
}
```

HIPPIEKICK commented on style.css:
```diff
@@ -0,0 +1,398 @@
+  font-weight: lighter;
+  text-align: justify;
+  line-height: 1.5;
+  color:grey
```
Don't forget to end your lines with semi colon

HIPPIEKICK commented on style.css:
```diff
@@ -0,0 +1,398 @@
+  flex-wrap: wrap;
+  justify-content: center;
+  font-family: 'Raleway', sans-serif;
+}
```
Nice flexboxing 💪

HIPPIEKICK commented on style.css:
```diff
@@ -0,0 +1,398 @@
+  padding: 10px;
+}
+
+.idea_1, .idea_2, .idea_3, .idea_4, .idea_5, .idea_6  {
```
Suggestions:
- Either give the HTML elements a generic class name and then look into the nth selector in CSS.
- Or give the HTML elements two class names where one is generic and one is element specific (and then put all generic styling under the generic class name and the specific under the specific)

HIPPIEKICK commented on style.css:
```diff
@@ -0,0 +1,398 @@
+  border-color:rgba(244, 136, 97, 0.488);
+  width:auto;
+  color: white;
+  cursor: pointer;
```
Nice!

HIPPIEKICK commented on style.css:
```diff
@@ -0,0 +1,398 @@
+.btn:hover {
+  background-color: rgb(244, 136, 97);
+  font-weight: bold;
+}
```
👌

HIPPIEKICK commented on code/style.css:
```diff
@@ -1,5 +1,319 @@
+  background-repeat: no-repeat;
+  background-size: 600px 350px;
+  border-radius: 5%;
+  padding: 15px;
```
Maybe some more padding on the sides to make it easier on the eye?

HIPPIEKICK replied:
Love the design on the cards in general!

HIPPIEKICK reviewed (commented):
Hi Rebecca!
Great job with this website, I really like your overall design, the color choices and the images. I think your code looks quite clean, although I gave some comments on how you can make it less repetitive. In web development, there's this principle called DRY (don't repeat yourself) that we should try to follow.

But as I said, really nice job 🔥

RebeccaBrogli committed:
Add files via upload

I added a Hamburger menu, however I failed to made it work (I need some java script).
I finished with the desktop mobile adjustments
I added a footer

RebeccaBrogli committed:
Add files via upload

I added a Feedback Form in the About me page.

RebeccaBrogli committed:
Add files via upload

Added footer icons (not linked yet), they only appear on the start site and not on the other sites (I don't know why). 

On the about me site: added a Feedback Box: Button, Text box, Mail box

RebeccaBrogli committed:
Add files via upload

I changed some styling on the feedback website. Used a texture instead of an input which allowed me to increase the text box size. I made each field responsive.

RebeccaBrogli committed:
Add files via upload

Projet 3: Spark conversations
Forum, style, and data sent between webpages and services. 

- Buttons (Link)
- Submit Button
- Text Fields
